README.md

```diff
@@ -28,19 +28,10 @@ See `https://ckeditor.com/latest/samples/toolbarconfigurator/index.html#advanced
 
 See [README-develop.md](README-develop.md) for information on setting up for development.
 
-## Todo
-Maybe we should use the composer-merge plugin from wikimedia, where we will have a composer-dev.json which overrides the
-main composer.json. In this dev composer json the repo to the core-bundle package is defined as a path, but in the main
-composer file the repo is defined as a vcs.
-
-The composer merge plugin should be added under the require-dev section in the main composer file, so it will only
-run when --no-dev is not present when running composer install.
-
-This will replace the initializing and updating of submodules when deploying to production, because the required kontrolgruppen
-package will be fetched from vcs instead.
-
 ## Release procedure.
 
+Make sure that the composer.json is using the correct version constraint for the core-bundle, and that the repository for the core-bundle is set to the vcs-type and the url is the git repository.
+
```

Comment:
Maybe add something like

```sh
composer config repositories.kontrolgruppen/core-bundle vcs git@github.com:aakb/kontrolgruppen-core-bundle.git
```
(do we also need `composer require 'kontrolgruppen/core-bundle:«some branch name»'` here?)

and/or something like

> Run `git checkout --patch composer.json` to revert your develop changes to `composer.json`.